azimuth_capi/models/v1alpha1/cluster.py

```diff
@@ -387,6 +387,10 @@ class ClusterStatus(schema.BaseModel, extra = "allow"):
         default_factory = dict,
         description = "The services for the cluster, indexed by service name."
     )
+    last_updated_timestamp: schema.Optional[dt.datetime] = Field(
+        default = None,
+        description = "Used to trigger the timeout of pending states"
+    )
```

Comment:
Minor nit, but I think the "timestamp" is not needed.

Also, would it make sense to actually make this a timestamp? It is not user-facing 🤷‍♂️

Reply:
yeah, true.